### DISCUSSION TITLE
Can somehow, instead of vscode.dev, open those scripts locally

### DISCUSSION THREAD

**Discussion by stdedos** [Tampermonkey/tampermonkey#1705] [Q&A]
Hello there,

I noticed this https://github.com/Tampermonkey/tampermonkey/issues/1654#issuecomment-1335815215 popping up recently!

I was wondering - would it be possible (once this feature lands on stable), to also use it with a locally-installed editor?

**Comment by derjanb** [comment]
Please try [TamperDAV](https://github.com/Tampermonkey/tamperdav#readme)
